### ISSUE TITLE
W3C implied endorsement

### ISSUE THREAD

**Issue by jwrosewell** [w3c/w3process#480]
All proposals and work unrelated to standards track will not be carried out under the W3C brand. An alternative brand will be created for the purpose and all community groups will operate under that brand with separate web sites, documents, repositories and resources.

This change is important to separate the process of experimentation (aka as incubation) from the W3C brand, which is well known in the industry.

This will prevent companies from using the W3C brand to convey legitimacy on proposals which may actually be anticompetitive, thus risking liability for the W3C and its members.

**Comment by chaals** [comment]
I strongly oppose this proposal. 

W3C's brand includes its long history of experimentation, its failures as well as its successes, and the environment it has created that allows for exploration of important technical and "techno-societal" issues as much as it includes the specifications it has produced that have successfully been applied as the underpinning standards of one of the world's most critical systems.

Given the broad scope of W3C, attempting to determine what is and what is not related to standards track work strikes me as a huge, unsustainable overhead that would merely serve to stifle discussion or limit it to those sufficiently well-resourced to manage an explosion in arguments that don't begin to approach the merits of the matter under discussion.

That said, I support the ongoing efforts made to be clear when and how much of W3C's endorsement is associated to something, because that endorsement is necessarily drawing on the collective reputations of its members.

**Comment by LJWatson** [comment]
> This change is important to separate the process of experimentation (aka as incubation) from the W3C brand, which is well known in the industry.

It is reasonable for activities that take place under the auspices of the W3C to be acknowledged as such. Fostering new ideas has long been a part of the the W3C has done - even CSS, SVG and ARIA were new ideas once.

I do think it is important to make the status of a document clear, but it seems to me that "Draft community report" in addition to the following text more than adequately does this:

> This specification was published by the Web Platform Incubator Community Group. It is not a W3C Standard nor is it on the W3C Standards Track.

**Comment by dwsinger** [comment]
I agree (and have already requested) that we it would be better if we used clearer styling to differentiate documents. For example, a final Recommendation should stand out as an agreed W3C document; the agreed working draft of a group (especially once it's at CR stage or later) should be differentiated from contributions or other individual documents. Community group work (including incubation) should be distinctly styled from W3C-membership group work (especially, documents that are chartered as part of WG work), so that Rec-track and explorations are clearly differentiated.

Is this what you mean?

**Comment by mnot** [comment]
FWIW - 

We see similar 'brand confusion' in the IETF, with many -- even reputable technical journalists -- portraying Internet Drafts (which anyone can publish without review) as IETF-sponsored (or even approved!) documents.

We've found that adding boilerplate and defining document status doesn't make a difference. 

Changing the design might help, if it's drastic enough; I've proposed Comic Sans in the past, and I don't think I was joking.

At the end of the day, many people look at two things to figure out who sponsors a document: the logo at the top and the domain of the URL it's hosted on. Changing both would help this kind of confusion -- most of the world's population doesn't understand the difference between a 'community group' and a 'working group'; they see the W3C logo at the top and w3c in the URL, and they assume it comes from the W3C.

So I'm sympathetic to the intent here. If the W3C wants to maintain value in the consensus it is based upon, it needs to clearly distinguish its output from its input. That doesn't mean that we should deny that a proposal is being made to the W3C, nor that it should disallow experimentation. I believe we can encourage proposals to be made in public and collaboratively by allowing them to be associated with the W3C, without allowing a proposal to confuse readers as to its status, taking into account the cognitive biases that might be in play (i.e., allowing users to be confused about draft status should be considered a dark pattern).

Concretely, I'd suggest:

* Coming up with a new logo for proposals
* Moving all CG and BG activity to a separate domain (e.g., `proposals.w3.org`; `w3c-proposals.org`)
* Moving all CG and BG activity to a separate GitHub org (e.g., `w3c-proposals`, `w3c-community-proposals`)

As you can probably tell, I think making the word 'proposal' prominent would help; other things might work too, but let's not hide the status behind jargon like 'community group'.

**Comment by dwsinger** [comment]
I'd like CG/BG work to be clearly as something that the W3C supports happening, and provides some light infrastructure for, but that somehow is at arm's length from W3C-endorsed work.

**Comment by jwrosewell** [comment]
It appears that we agree that there is an issue of confusion in the public mind when things are put to W3C. For example a “draft discussion document” can easily be confused with a “proposal for a standard” or otherwise thought of as a documents that is legitimately being debated as the basis for agreement.  As discussed, even the use of the words “Internet Drafts” which anyone can publish without review are confused by journalists with something that IETF endorses, and it is recognized that this can cause brand confusion. So I think we all agree that “Brand Endorsement” is an increasing online problem.  

If we agree on the issue of brand confusion – what is to be done to clear up the confusion?

Two alternatives have been put forward – better labelling of documents and some way that members can decide that a matter is suitable for discussion; or both? Better labelling won’t do anything for the confusion in the public mind as people outside W3C won’t know what type of document or proposal is being discussed and will naturally expect that only things suitable for discussion are on the agenda.

This issue relates specifically to implied endorsement beyond the W3C. It could be combined with [474](https://github.com/w3c/w3process/issues/474), [475](https://github.com/w3c/w3process/issues/475), [476](https://github.com/w3c/w3process/issues/476), [477](https://github.com/w3c/w3process/issues/477) & [478](https://github.com/w3c/w3process/issues/478) if there was a willingness to identify a combined solution to this group of issues.
